src/DIRAC/Resources/Cloud/CloudEndpoint.py

```diff
@@ -0,0 +1,573 @@
+###########################################################
```

Comment:
We generally do not use this kind of comments, can be scrapped. Here and at other places

Reply:
I've removed all comments of this style that I could find.